tests/runmode_test/test.yaml

```diff
@@ -0,0 +1,10 @@
+requires:
+  min-version: 7.0.0
+  pcap: false
+
+checks:
+  - shell:
+      args: suricata
+      expect: 1  # Expect an exit code of 1, indicating an error
+      expect-output: "ERROR: Missing runmode"  # Verify that the error message is present
+
```

Comment:
We don't have a check option like that in Surica-verify, you could have a look at https://github.com/OISF/suricata-verify/pull/1431/files to see how to check for log output error messages in SV tests :)